### PR TITLE
x/debug: add sanity check shares

### PR DESCRIPTION
**Pull request by ilovers**
- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/okex/okchain/blob/develop/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Reviewed `Files changed` in the github PR explorer

______

For Admin Use:
- Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- Reviewers Assigned
- Squashed all commits, uses message "Merge PR #XYZ: [title]" ([coding standards](https://github.com/okex/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: x/staking/keeper/sanity_check.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// SanityCheck checks validator shares
+func (k Keeper) SanityCheck(ctx sdk.Context) error {
```
Comment (xiangjianmeng) [inline x/staking/keeper/sanity_check.go:12]: Maybe SanityCheck could reuse `DelegatorAddSharesInvariant`.

Comment (ilovers) [reply x/staking/keeper/sanity_check.go]: Good idea!

File: x/debug/types/keys.go [hunk 2]
```diff
@@ -13,6 +13,7 @@ const (
 	// QuerierRoute is the querier route for the debug module
 	QuerierRoute = ModuleName
 
-	DumpStore    = "dump"
-	SetLogLevel  = "set-loglevel"
+	DumpStore   = "dump"
+	SetLogLevel = "set-loglevel"
+	SanityCheck = "sanity-check"
```
Comment (zhongqiuwood) [inline x/debug/types/keys.go:18]: use sanity-check-shares

Comment (ilovers) [reply x/debug/types/keys.go]: done

**Pull request merged**